packages/cli/src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './ceramic-cli-utils.js'
 export * from './ceramic-daemon.js'
 export * from './daemon-config.js'
 export * from './s3-store.js'
+export * from './__tests__/make-did.js'
```

Comment:
this feels possibly like bad practice?  @ukstv let me know if you have ideas for a better way I can share this testing utility function between the `cli` and `stream-tests` packages.  Or maybe this approach is fine?